lib/internal/url.js

```diff
@@ -1320,8 +1323,8 @@ function getPathFromURLWin32(url) {
       var third = pathname.codePointAt(n + 2) | 0x20;
       if ((pathname[n + 1] === '2' && third === 102) || // 2f 2F /
           (pathname[n + 1] === '5' && third === 99)) {  // 5c 5C \
-        return new TypeError(
-          'Path must not include encoded \\ or / characters');
+        return new errors.TypeError('ERR_INVALID_FILE_URL_PATH',
+                                    'not include encoded \\ or / characters');
```

Comment:
It's a bit weird to put the verbs in separate places...maybe the `must` part can be left out in the error message format and written here?